torch/_torch_docs.py

```diff
@@ -3561,6 +3561,38 @@ def merge_dicts(*dicts):
     RuntimeError: bool value of Tensor with no values is ambiguous
 """.format(**common_args))
 
+add_docstr(torch.kron,
+           r"""
+kron(input, other, *, out=None) -> Tensor
+
+Computes the Kronecker product of :attr:`input` and :attr:`other`.
+
+If :attr:`input` is a :math:`(n \times m)` tensor, :attr:`other` is a
+:math:`(k \times l)` tensor, the result will be a :math:`(n*k \times m*l)` tensor.
```

Comment:
This documentation is great. What about adding one more sentence and a mathematical statement explaining what, exactly, the Kronecker product is? I thought the first picture in its Wikipedia article (see https://en.wikipedia.org/wiki/Kronecker_product) was small and descriptive.

Reply:
Sure, I've added the definition as in Wikipedia. I hope it renders correctly